Project.toml

```diff
@@ -16,4 +16,4 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 AbstractMCMC = "2, 3, 4"
 DensityInterface = "0.4"
 Setfield = "0.7.1, 0.8"
-julia = "1.6"
+julia = "~1.6.6, 1.7.3"
```

Comment:
Hei @devmotion, you are more versioning wizard than I -- basicallywe want to disallow 1.7.x below patch version 3 here, as deepcopy is buggy in those (#66). Is this a good bound for that?

Reply:
Probably fine to ignore this issue since we haven't even exported the `GraphPPL` module yet.